modules/services/agentless-scanning/main.tf

```diff
@@ -186,7 +187,7 @@ resource "aws_iam_role" "agentless" {
 resource "aws_iam_policy_attachment" "agentless" {
   count = var.deploy_global_resources ? 1 : 0
 
-  name       = "sysdig-agentless-host-scanning"
+  name       = var.name
```

Comment:
do you know why we don't have name for aws_kms_key and aws_kms_replica_key resource?

Reply:
doesn't look like it's available it seems.